src/main/java/bio/terra/cli/command/shared/options/BqDatasetLifetime.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.cli.command.shared.options;
+
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the default lifetime options for `terra resource` commands that
+ * handle Big Query Datasets.
+ *
+ * <p>This class is meant to be used as a @CommandLine.Mixin.
+ */
+public class BqDatasetLifetime {
+
+  @CommandLine.Option(
+      names = "--default-partition-lifetime",
+      description =
+          "The default lifetime, in seconds, for partitions in newly created partitioned tables. This flag has no minimum value. Specify 0 to remove the existing expiration time. Any partitions in newly created partitioned tables are deleted this many seconds after the partition's UTC date. This value is applied if you do not set a partition expiration on the table when it is created.")
+  private Integer defaultPartitionLifetime;
+
+  @CommandLine.Option(
+      names = "--default-table-lifetime",
+      description =
+          "The default lifetime, in seconds, for newly created tables. The minimum value is 3600 seconds (one hour). The expiration time evaluates to the current UTC time plus this value. Specify 0 to remove the existing expiration time. Any table created in the dataset is deleted this many seconds after its creation time. This value is applied if you do not set a table expiration when the table is created.")
+  private Integer defaultTableLifetime;
```

Comment:
This does work, but it necessitates the user inputting durations using [ISO-8601 duration format](https://en.wikipedia.org/wiki/ISO_8601#Durations), which means that:
```
--default-table-lifetime 3600
```
becomes:
```
--default-table-lifetime PT3600S
```
or:
```
--default-table-lifetime PT1H
```

My initial read is that Jackson will parse to/from this format when ser/des a `Duration`, so that could be made consistent.  But I'm not sure we want to push the user to have to look up ISO-8601 for a case where seconds are probably sufficient.  Interested in others' thoughts.

Reply:
sounds good. I think it's worth suffixing with the 'seconds' or 'sec' or 's' time unit then anywhere we're using an integer for a number of seconds.

Reply:
I take the point on adding seconds suffix.  I feel like `--default-table-lifetime-seconds` is getting a little unwieldy, what do folks think about dropping the `default` in the cli switch (`--table-lifetime-seconds`)?  I'm on the fence because both concepts seem notable.

Reply:
You can leave the command name but change the variable name. I'm fine leaving the command as unspecified units asking the user to read the description (which they're probably doing anyway), but `Integer defaultTableLifetime` is just ambiguous, and not all of them on all the classes are commented.

Reply:
+1 for keeping the command name the same, to match what `bq` CLI does.